mgmtsystem/azure.py

```diff
@@ -68,16 +69,20 @@ def pre_script(self):
         Get-AzureRmSubscription -SubscriptionId \"{}\" -TenantId \"{}\" | Select-AzureRmSubscription
         """.format(self.ui_username, self.ui_password, self.subscription_id, self.tenant_id))
 
-    def run_script(self, script, ignore_error=False):
+    def run_script(self, script, _tries=3, ignore_error=False):
```

Comment:
please put your param to the end of param list. and I think it is not private so, underscore isn't needed.